1-controllers/comments-controller.js

```diff
@@ -11,14 +15,30 @@ exports.getCommentByArticleId = (req, res, next) => {
 };
 
 exports.PostComment = (req, res, next) => {
-    const { article_id } = req.params
-    const {username, body} = req.body
-    
-    insertComment(username, body, article_id)
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+
+  insertComment(username, body, article_id)
     .then((comment) => {
-        
-        res.status(201).send({ comment })
+      res.status(201).send({ comment });
+    })
+    .catch(next);
+};
+
+exports.deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+
+  if (Number(comment_id)) {
+    deleteCommentbyId(comment_id)
+    .then((deletion) => {
+      const deletedData = deletion[0].rows[0];
+      if (!deletedData) {
+        res.status(404).send({ message: "Comment Id Not Found" });
+      } else {
+        res.status(204).end()
+      } 
     })
-    .catch(next)
-    
-}
+  } else {
+    res.status(400).send({message: "Bad Request : Wrong Data Type"})
+  }
+};
```

Comment:
Your if/else here likely isn't necessary - if you data type is wrong PSQL would throw an error